schemas/session-replay/mobile/webview-wireframe-update-schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/webview-wireframe-update-schema.json",
+  "title": "WebviewWireframeUpdate",
+  "type": "object",
+  "description": "Schema of all properties of a WebviewWireframeUpdate.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-update-schema.json"
+    },
+    {
+      "required": ["type", "webViewId"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "webview",
+          "readOnly": true
+        },
+        "webViewId": {
```

Comment:
this field is read only so it should not be updated. However, even though the wireframe.id would be sufficient to identify the shape to update, the `webViewId` will help up identifying webview container updates out of all the shape wireframe updates.

Reply:
Talking with Bastien, he suggested `bridgeId` as naming. This way we are agnostic from whether it's a webview or an iFrame(in the future). I also think it's a nice option. Wdyt?